src/LaunchDarkly.EventSource/Configuration.cs

```diff
@@ -2,29 +2,48 @@
 using System.Collections.Generic;
 using System.Net.Http;
 using System.Text;
-using System.Threading;
 using LaunchDarkly.Logging;
 
 namespace LaunchDarkly.EventSource
 {
     /// <summary>
     /// An immutable class containing configuration properties for <see cref="EventSource"/>.
     /// </summary>
+    /// <seealso cref="EventSource.EventSource(Configuration)"/>
     /// <seealso cref="ConfigurationBuilder"/>
     public sealed class Configuration
     {
-        #region Types
+        #region Constants
 
-        public delegate HttpContent HttpContentFactory();
+        /// <summary>
+        /// The default value for <see cref="ConfigurationBuilder.DelayRetryDuration(TimeSpan)"/>:
+        /// one second.
+        /// </summary>
+        public static readonly TimeSpan DefaultDelayRetryDuration = TimeSpan.FromSeconds(1);
 
-        #endregion
+        /// <summary>
+        /// The maximum value for <see cref="ConfigurationBuilder.DelayRetryDuration(TimeSpan)"/>:
+        /// 30 seconds.
+        /// </summary>
+        public static readonly TimeSpan MaximumRetryDuration = TimeSpan.FromSeconds(30);
```

Comment:
It seems a bit inconsistent to call this `MaximumRetryDuration` rather than `MaximumDelayRetryDuration`. Do you think this is something worth changing?

Reply:
I'd been trying to avoid making breaking changes that were _only_ for better naming, but I agree that these are weirdly named— both of them. "DelayRetryDuration" is weird because it looks like it should mean "duration of the retry of the delay", when it's really "duration of the delay before the retry". And "MaximumRetryDuration" is wrong because it's not a duration of a retry.

Reply:
And I think "Duration" is redundant in both cases because a delay that is represented by a TimeSpan is, by definition, a duration.

Reply:
Huh, I was actually confused about what these were doing, probably partially because of the names. 

I was thinking this was the default value of a configurable property that would control the maximum delay time of the backoff. While it is the maximum backoff delay time, it is not a configurable property, and also functions as the maximum value of the initial retry delay.

Maybe this isn't the right PR to bring this up in, but this behavior doesn't make a lot of sense to me. I don't see the value in limiting the maximum value of the initial backoff delay. It would make more sense to me to just allow configuring the maximum delay of the backoff.

If this doesn't makes sense right now or I'm missing something, then maybe we could just include a line in the documentation comment for `MaximumRetryDuration` to mention that it is also the maximum delay after backoffs as well.

Reply:
> I don't see the value in limiting the maximum value of the initial backoff delay. It would make more sense to me to just allow configuring the maximum delay of the backoff.

That's reasonable, and it corresponds to how some of our other EventSource implementations are configured.